### DISCUSSION TITLE
How do you set reminders/verification on discussions?

### DISCUSSION THREAD

**Discussion by BekahHW** [community/community#30690] [Discussions]
How do you add the reminder to search for similar discussions and the checkbox to say that you searched to discussions?

![image](https://user-images.githubusercontent.com/34313413/186496591-5a8e7235-edd9-4c73-b823-88293b8739a6.png)

![image](https://user-images.githubusercontent.com/34313413/186496617-76961e74-444a-42bf-861c-3c3ee32df85b.png)

**Comment by ghost** [comment]
Hi! BekahHW Great idea!
